content/docs/join_us/recruitment_FAQs.md

```diff
@@ -33,16 +33,32 @@ Yes, the positions are open to applicants outside the UK and our HR team ([HR@tu
 
 Members need to be based in the UK.
 However, you do not need to be in the London office every day: almost all of our work is done in a hybrid manner, and we have team members as far away as Scotland and the South West of England.
+It is (at least on paper) expected that you will be in the office four days a month, and there are two events through the year where we encourage everyone to come in: the Away Days (two days in February) and Hack Week (one week in June).
 
 ### What would be the starting date?
 
-We are keen to grow the team so would prefer if people can start within 3 months of an offer being made and would be happy for successful candidates to start sooner.
-However, we are looking to recruit multiple people and we are hiring for the long term, so we can support later starting dates if a successful candidate's circumstances require it.
+In the past, we have preferred that people start within three months of an offer being made; we are also happy for successful candidates to start sooner.
+<!-- TODO: Check shortest and longest time people have joined post-offer -->
```

Comment:
@IFenton @phinate I think you all might be good representatives of short and long respectively? For me it was almost exactly the full 3 months between offer and start.

Reply:
```suggestion
We generally prefer that people start within three months of an offer being made.
However, if necessary, we can also accommodate shorter or longer periods between the offer and starting; for previous candidates this time has ranged from 1 month to ~ 6 months.
```

Would this be fair? I think it's good to have an exact range because that tells applicants whether they should bother applying.